speechbrain/utils/fetching.py

```diff
@@ -51,56 +46,265 @@ class FetchFrom(Enum):
-    save_filename : str
-        The filename to use for saving this file. Defaults to filename if not
-        given.
-    use_auth_token : bool (default: False)
+        Directory path where the file will be reachable (unless using
+        `LocalStrategy.NO_LINK`, in which case it may or may not end up in this
+        directory).
+    overwrite : bool (default: `False`)
+        If `True`, always overwrite existing savedir/filename file and download
+        the file over again, even if it was unchanged.
+        If False (the default), if savedir/filename exists, assume it is
+        correct and don't download/relink.
+        Note that Huggingface local cache is always used - with overwrite=True
+        you may just relink from the local cache.
+    allow_updates : bool (default: `True`)
+        If `True`, for a remote file on HF, check for updates and download newer
+        revisions if available.
+        If `False`, when the requested files are available locally, load them
+        without fetching from HF.
+    allow_network : bool (default: `True`)
+        If `True`, network accesses are allowed. If `False`, then remote URLs
+        or HF won't be fetched, regardless of any other parameter.
+    save_filename : str, optional (default: `None`)
+        The filename to use for saving this file. Defaults to the `filename`
+        argument if not given or `None`.
+    use_auth_token : bool (default: `False`)
         If true Huggingface's auth_token will be used to load private models from the HuggingFace Hub,
         default is False because majority of models are public.
-    revision : str
-        The model revision corresponding to the HuggingFace Hub model revision.
-        This is particularly useful if you wish to pin your code to a particular
-        version of a model hosted at HuggingFace.
-    huggingface_cache_dir: str
-        Path to HuggingFace cache; if None -> "~/.cache/huggingface" (default: None)
+    revision : str, optional (default: `None`)
+        The model revision corresponding to the HuggingFace Hub model revision,
+        in the form of a Git revision (branch name, tag, commit hash).
+        This allows to pin your code to a particular version of a model hosted
+        on HuggingFace.
+        When changing the revision while local files still exist,
+        `allow_updates` must be `True`.
+    huggingface_cache_dir: str, optional (default: `None`)
+        Path to HuggingFace cache; if `None`, the default cache directory is
+        used: `~/.cache/huggingface` unless overridden by environment variables.
+        See `huggingface_hub documentation <https://huggingface.co/docs/huggingface_hub/guides/manage-cache#manage-huggingfacehub-cache-system>`_
+        Ignored if the local strategy is `LocalStrategy.COPY_SKIP_CACHE`.
+        (default: None)
+    local_strategy : LocalStrategy, optional
+        Which strategy to use to deal with files locally. (default:
+        `LocalStrategy.SYMLINK`)
```

Comment:
This default was updated to `NO_LINK`, documentation should be updated to match.
```suggestion
        `LocalStrategy.NO_LINK`)
```